### PR TITLE
[YOLOS] Minor fixes

### PR DESCRIPTION
**Pull request by NielsRogge**
# What does this PR do?

This PR updates a Copied from statement for YolosFeatureExtractor, and removes 2 methods which aren't necessary for the model (as there is no segmentation head model).

### CODE REVIEW

File: docs/source/en/model_doc/yolos.mdx [hunk 1]
```diff
@@ -43,9 +43,7 @@ This model was contributed by [nielsr](https://huggingface.co/nielsr). The origi
 [[autodoc]] YolosFeatureExtractor
     - __call__
     - pad
-    - post_process
-    - post_process_segmentation
-    - post_process_panoptic
+    - post_process_object_detection
```
[the remark this answers is not in the document]
Comment (sgugger) [reply docs/source/en/model_doc/yolos.mdx:46]: Yes and no!

**Review by sgugger — APPROVED** [review]
Thanks a lot!

File: docs/source/en/model_doc/yolos.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (alaradirik) [reply docs/source/en/model_doc/yolos.mdx:46]: +1 I removed the deprecated methods from docs and added the new ones for the segmentation models

**Review by alaradirik — APPROVED** [review]
LGTM!

**Pull request merged**